comfy_cli/command/generate/app.py

```
@@ -0,0 +1,370 @@
+    try:
+        api_key = client.resolve_api_key(meta.get("api-key") if isinstance(meta.get("api-key"), str) else None)
+    except client.ApiError as e:
+        rprint(f"[bold red]{e}[/bold red]")
+        raise typer.Exit(code=1)
+
+    timeout_raw = meta.get("timeout", "300")
+    try:
+        timeout = float(timeout_raw) if isinstance(timeout_raw, str) else 300.0
+    except ValueError:
+        rprint(f"[bold red]--timeout: expected number, got {timeout_raw!r}[/bold red]")
+        raise typer.Exit(code=1)
```

Pylint (4.0.5)

[warning] Consider explicitly re-raising using 'except ValueError as exc' and 'raise typer.Exit(code=1) from exc'

(W0707)